num_rob: 2
sph0: sphero_sk9495
sph1: sphero_sk1270